packages/utils/src/__tests__/checkValueSatisfiesCondition.test.js

```diff
@@ -1,114 +1,111 @@
-import { expect } from 'chai';
 import { checkValueSatisfiesCondition } from '../checkValueSatisfiesCondition';
 
 describe('checkAgainstConditions', () => {
   describe('checkValueSatisfiesCondition()', () => {
     it('should throw an error if given an unknown operator', () => {
-      expect(() => checkValueSatisfiesCondition('hello', { operator: 'NOT_VALID' })).to.throw(
-        "Unknown operator: 'NOT_VALID'",
-      );
+      expect(() => checkValueSatisfiesCondition('hello', { operator: 'NOT_VALID' })).toThrowError("Unknown operator: 'NOT_VALID'");
     });
 
     it('should correctly check if a value satisfies a plain condition', () => {
-      expect(checkValueSatisfiesCondition('hello', 'hello')).to.be.true;
-      expect(checkValueSatisfiesCondition(0, 0)).to.be.true;
-      expect(checkValueSatisfiesCondition(true, true)).to.be.true;
-      expect(checkValueSatisfiesCondition(false, false)).to.be.true;
-      expect(checkValueSatisfiesCondition(null, null)).to.be.true;
+      expect(checkValueSatisfiesCondition('hello', 'hello')).toBe(true);
```

Comment:
This file seems like a great candidate for using parameterized tests. Let's do this in another PR though!

Reply:
You've already done it! 🙌